src/blog/posts/service.py

```diff
@@ -19,7 +19,7 @@ def get(post_id: int) -> t.Optional[Post]:
     for post in post_table:
         if post["id"] == post_id:
             searched_post = post
-
+    breakpoint()
```

Comment:
You need to remove breakpoints before committing the changes.

Reply:
I forget to delete it. Now it's fixed.